src/bootstrap.js

```diff
@@ -1,21 +1,21 @@
-import 'polyfill';
-import 'asset/style/screen.scss';
 import 'modernizr';
-import 'settings';
 
 import Vue from 'vue';
-import filter from 'filter';
-import directive from 'directive';
-import component from 'component';
-import getRouter from 'router';
-import getStore from 'store';
-import startUp from 'control/startUp';
-import getLocaleConfig from 'config/localeConfig';
 import VueI18nManager from 'vue-i18n-manager';
 import { sync } from 'vuex-router-sync';
-import setupInjects from 'util/setupInjects';
-import localeLoader from 'util/localeLoader';
-import App from 'App';
+import './asset/style/screen.scss';
+import './polyfill';
```

Comment:
@flut1 It's also better to move this all the way to the top because you never know the code that is executed before this import. It may need a polyfill in some browsers.

Reply:
I think the safer way to do this is by adding it to the entry. So removing the import, and then changing the webpack config to
```
  entry: {
    app: ['./src/polyfill/index.js', './src/bootstrap.js'],
  },
```

Reply:
Yes that is also a possible solution